### ISSUE TITLE
Imported settings do not persist across launches (Windows 10)

### ISSUE THREAD

**Issue by jaustinhughey** [lukesampson/concfg#8]
I suspect this may be a Windows 10 specific issue, but I'm quite the novice with Power Shell; Luke, I'm hoping you can shed some light on this :)

1. Launch PowerShell from start menu (either cmd+x and arrow up, or cmd, type "powershell" and hit enter once it's found)
2. ```concfg import solarized small```
3. "...would you like to find and remove them?" Y
4. stuff -- ```overrides removed```
5. "...open new console to see the changes?" Y

New console opens. Nice! But now I exit the console and the previous console (close all Power Shell windows) and then re-open powershell. And then I'm back to the default colors.

The first time you do this, it'll find and remove some overrides, but subsequent times, it can't seem to find any overrides to remove so it just says they're removed and has no further output between those commands:

```
~ $ concfg import solarized small
console settings were imported from solarized
console settings were imported from small

overrides in the registry and shortcut files might interfere with your concfg
settings.
would you like to find and remove them? (Y/n): y
overrides removed
would you like to open a new console to see the changes? (Y/n): y
~ $
```

Any ideas what I can do to get this to persist? Thanks!

**Comment by jaustinhughey** [comment]
Wow - I think I just figured this out myself, very sorry for the noise!

I was closing windows by hitting the UI element (the X); instead, type "exit" and hit enter. Then re-launch. Now it seems to have persisted. Sweet!

**Comment by ChadFong** [comment]
I'm having the same issue on Windows 10, and using the `exit` command does not resolve it.

**Comment by jaustinhughey** [comment]
Chad, in my case I had launched the new console through concfg and hit "y" to say "yes open a new one so I can see changes", and from THAT console window I typed "exit". Give that a try, maybe after a re-installation?

**Comment by ericdvb** [comment]
Seeing the  same issue in Windows 10. Just tried out using exit to close out the auto opened console, but no dice.

**Comment by lukesampson** [comment]
Have you tried using `sudo concfg clean`? You might need to run `scoop install sudo` first. This cleans saved settings from the registry and .lnk files, but it doesn't really work unless you `sudo`. After that, you might need to run `sudo concfg import [theme]` to be doubly sure.

**Comment by ericdvb** [comment]
Just tried that out - it doesn't seem to change anything from the previously described scenario. I've noticed that the the "default" colors (shown when I click the powershell icon at top left, then click "defaults") are the solarized colors after running `sudo concfg import solarized`, and they persist across new windows in the "default" dialog. However the actual console color scheme for any new powershell windows adheres to the colors in the "properties" menu. I wonder if this is because at some point I changed the colors in the "properties" dialog?

**Comment by ericdvb** [comment]
By the way, thank you for your work on this! It's a pretty sweet tool.

**Comment by lukesampson** [comment]
I think I found the problem—I was always opening powershell from the taskbar so I didn't notice, but the start menu .lnks are in a different place for Win10, and concfg wasn't cleaning those. It should be fixed now if you run `scoop update concfg` and then `sudo concfg import [theme]`.

**Comment by ericdvb** [comment]
That was impressively fast - thank you! Also just wanted to say the ux of scoop is a vast improvement over chocolatey - nice work all around :+1:

**Comment by lukesampson** [comment]
Great! Thanks man, glad you like it :smile:

**Comment by AdrianoCahete** [comment]
I have the same problem at Win 8.1 and Powershell 5.
Tried all that commands here and nothing works. :(

Any way to supress questions and running for profile configs?

**Comment by chrisfillmore** [comment]
Adriano, I was just reading this thread and had the same problem, and I was able to solve it. Find the right PowerShell shortcut here:

C:\ProgramData\Microsoft\Windows\Start Menu\Programs\Accessories\Windows PowerShell

In my case (I'm running Windows 7, 64-bit), I needed to modify "Windows PowerShell" shortcut.

Right-click on the shortcut, Properties, Security tab, Edit (you'll need admin access here). I changed the permissions so that "Everyone" could modify the file.

Then I launched PowerShell again, ran `concfg import solarized`, and was able to remove the overrides successfully.

Afterward I changed permissions on the shorcut back to their defaults.
